probert/multipath.py

```diff
@@ -43,7 +44,7 @@ def _extract_mpath_data(cmd, show_verb):
     for line in data.splitlines():
         mp_dict = None
         try:
-            field_vals = line.split()
+            field_vals = line.split(MP_SEP)
```

Comment:
It's not something that needs to be addressed in this PR, but this does reveal a shortcoming in the current interface of this function: it relies on its callers specifying an appropriate `format` parameter in `cmd`, else it will produce incorrect output. And, looking at it, this is also true of `show_verb`. The construction of `cmd` could probably move into this function, so it takes just `show_verb` and `format_fields` as parameters, where `format_fields` is the list of format fields before being joined with `MP_SEP`.

Reply:
I've added issue #85 to track this work.